crates/ide-assists/src/utils.rs

```diff
@@ -94,16 +94,44 @@ pub fn filter_assoc_items(
     sema: &Semantics<'_, RootDatabase>,
     items: &[hir::AssocItem],
     default_methods: DefaultMethods,
+    ignore_hidden: bool,
 ) -> Vec<InFile<ast::AssocItem>> {
+    // FIXME : How to use the closure below  this so I can write sth like
```

Comment:
You can't, closures with extra generic parameters are not a thing (`impl Trait` in parameter position desugars to an unknown generic parameter)

Reply:
the repetition here is small though so its fine